.changeset/ten-candles-call.md

```diff
@@ -0,0 +1,9 @@
+---
+'@backstage/core-components': minor
```

Comment:
Think we might have missed these? Should be a `patch` 👍

Reply:
ah, ok 😬